sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/param_flow.js

```diff
@@ -157,16 +160,22 @@ angular.module('sentinelDashboardApp').controller('ParamFlowController', ['$scop
           paramFlowItemList: [],
           count: 0,
           limitApp: 'default',
+          controlBehavior: 0,
+          durationInSec: 1,
+          burstCount: 0,
+          maxQueueingTimeMs: 0,
           clusterMode: false,
           clusterConfig: {
-            thresholdType: 0
+            thresholdType: 0,
+            fallbackToLocalWhenFail: true,
```

Comment:
Here's an extra comma

Reply:
Yes javascript allows extra commas in dict/array in defining.

Reply:
Yes it's allowed in JavaScript.

Reply:
Thanks! Got it.